### ISSUE TITLE
Dark Mode

### ISSUE THREAD

**Issue by Snnackss** [Hubro/clipface#19]
Even just a pitch black AMOLED style mode would be great. Theatre mode is good, but Clipface still has bright white elements. I have used browser extensions in the past, but quit using them because sometimes they just start bugging out. Honestly, it doesn't even need to be a toggle on the webpage. Just something you can type into the docker run arguments when starting Clipface.

**Comment by Hubro** [comment]
Yep, I should definitely add this. I'm an ardent user of Dark Reader so the lack hasn't really bothered me 😁

**Comment by Snnackss** [comment]
Hey Hubro! I threw together this dark mode concept the other day, but before I completed it I realized all I had to do was load up Dark Reader or something to get an idea of what it should look like lol

Anyways, instead of scrapping it I think I will share it here https://imgur.com/plVBDCF

Honestly, if it is easier, having the ENTIRE background FULL AMOLED BLACK would also work with white text, but this is just an idea.

Here are the colors:
Main background: #232323
Header and footer background: #111111
White font: #ffffff
Grey font: #a3a3a5
Blue highlight: #3da8fa
